tools/docker/env/Dockerfile

```diff
@@ -112,6 +112,19 @@ RUN chmod 0777 /syzkaller
 ENV SOURCEDIR_FUCHSIA /syzkaller/fuchsia
 ENV SOURCEDIR_NETBSD /syzkaller/netbsd
 
+# Latest Rust compiler.
```

Comment:
Is it "latest" if we're pinning it to 1.86.0 (latest is 1.87). Perhaps "# Rust toolchain suitable for kernel builds."

Reply:
Actually, I was about to install `1.87`, but it was apparently released on that same day and the rustup script was still unable to download it :)

Updated the version and the comment. PTAL.